app/components/post_draft/post_input/post_input.tsx

```diff
@@ -349,6 +356,7 @@ export default function PostInput({
             textContentType='none'
             value={value}
             autoCapitalize='sentences'
+            nativeID={testID}
```

Comment:
To me is not clear why we are using the testID as the nativeID

Reply:
KeyboardGestureArea uses textInputNativeID to identify the text input. On iOS, this must match the nativeID of the TextInput for interactive keyboard dismissal to work.

https://kirillzyusko.github.io/react-native-keyboard-controller/docs/api/views/keyboard-gesture-area#textinputnativeid-

Reply:
I know and understand how it works, is just having the nativeId=testID what is bothering me, is just that, they can be the same id just that it seems weird because of the name of the var